docs/adr/0022-flow-bit-rate-properties.md

```diff
@@ -0,0 +1,189 @@
+The HLS definition is extended to include single segments above 1.5 the target segment duration to handle Flows that don't follow the HLS retrictions for segment durations, e.g. segmentation optimised for best quality, with variable and potentially large GOP sizes etc..
+
+The `max_bit_rate` could be calculated from the segments that are present in the TAMS.
+The value could be an expectation of what could be present in the TAMS (e.g. derived from an encoder / muxer setting for a live ingest).
+The value could be an approximation, ideally an upper bound (e.g. the bit rate from the H.264 `hrd_parameters` plus some container overhead approximation).
+The value could change over time as segments become available or are removed from TAMS.
+
+* Good, provides bit rate information for TAMS users that access essence via segments
+* Bad, the information provided by the Flow is incomplete and doesn't allow the bit rate to be used to decide how much segment data needs to be buffered in the receiver for example
+* Bad, it doesn't directly provide essence bit rate information for users of TAMS where the essence in other contexts is not transmitted in the same segments as TAMS
+
+### Option 3b: Define segment bit rates with target segment duration
+
+In [Option 3a](#option-3a-define-segment-bit-rates) the `max_bit_rate` on it's own doesn't provide sufficient information to calculate the size or duration of the receiver buffer needed to allow for continuous playback (under ideal network conditions).
+The descriptions and calculations from SCTE 214-1 section 10.3 provides insights to what information could be provided.
+
+A [_tams_segmentation_rate](../appnotes/0003-tag-names#_tams_segmentation_rate) Flow tag is currently used in some applications to provide a target segment rate for the Flow.
+This segment rate provides the target segment duration (1 / segment rate) that is used to calculate the `max_bit_rate`.
+The proposal is to add a `segment_duration` (rational, seconds) property that specifies the target segment duration for the Flow.
+A duration is used rather than a rate to avoid confusion with bit and frame rates and to match the descriptions and properties in the HLS and DASH specifications.
+A rational type allows more precise values to be set (e.g. to match frames rates), avoiding potential rounding issues when doing calculations over long periods of time.
+
+The description in SCTE 214-1 section 10.3 includes a formula for calculating the buffer size required for continuous playback.
+The buffer size is defined as `1.1 * MSR[R] * SDmax`.
+`MSR[R]` in the TAMS context is `max_bit_rate`, `SDmax` is the maximum segment duration and `1.1` adds a 10% overhead for additional event data.
+The proposal is to add a `max_segment_duration` (floating point, seconds) property that provides the maximum segment duration (`SDmax`) for the Flow.
```

Comment:
NIT: Could the use of float here be confusing? A quick look at the current TAMS spec, I don't think we use floats elsewhere. We seem to exclusively use ints and fraction objects.

Reply:
I was going for simplicity here as this property doesn't need to be as precise as `frame_rate` for example. However, I've changed it to a fraction object for consistency.